### ISSUE TITLE
ESP32可以同时作为网卡和普通单片机么？

### ISSUE THREAD

**Issue by iBigDane** [espressif/esp-hosted#45]

**Comment by MangeshMalusare** [comment]
Yes, It is possible to use it that way. Please note that wifi, bt/ble, SDIO, SPI and UART on ESP32 will be used by ESP-Hosted solution. So those components won't be available for other purposes.
Please let us know if you need more information.

Thanks,
Mangesh
